jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -34,7 +34,7 @@ describe('EntityDetails', function() {
   it('can be rendered', function() {
```

Comment:
I don't think so, there are three possible outputs from the render function and only a single test below. Take a look at how Huw did the tests in his branch here https://github.com/juju/juju-gui/pull/891/files he tests the full output splitting up the components into many smaller components to make it easier to test.